### ISSUE TITLE
PieChart label is not show

### ISSUE THREAD

**Issue by weue** [apache/echarts#14139]
### Version
5.0.1

### Steps to reproduce
Use the example to modify values
https://echarts.apache.org/examples/zh/editor.html?c=pie-simple
when i use this data, the second label is disappear
```javascript
data: [
  {value: 940, name: '搜索引擎'},
  {value: 660, name: '直接访问'},
  {value: 80, name: '邮件营销'},
  {value: 80, name: '联盟广告'}
]
```

### What is expected?
Display all labels normally

### What is actually happening?
The second label is disappear

**Comment by Ovilia** [comment]
I cannot reproduce "the second label is disappear".

**Comment by weue** [comment]
![image](https://user-images.githubusercontent.com/28719349/106247803-03c14880-624b-11eb-84e5-5a34a0e39fd6.png)
Use this data
`data: [
    {value: 940, name: '搜索引擎'},
    {value: 660, name: '直接访问'},
    {value: 80, name: '邮件营销'},
    {value: 80, name: '联盟广告'}
  ]`

**Comment by weue** [comment]
> 我无法重现“第二个标签消失”。




> I cannot reproduce "the second label is disappear".

![image](https://user-images.githubusercontent.com/28719349/106248064-6b779380-624b-11eb-8f93-13cc4add5406.png)
Use this data
`data: [
  {value: 940, name: '搜索引擎'},
  {value: 660, name: '直接访问'},
  {value: 80, name: '邮件营销'},
  {value: 80, name: '联盟广告'}
]`

**Comment by weue** [comment]
> I cannot reproduce "the second label is disappear".

![image](https://user-images.githubusercontent.com/28719349/106248299-c315ff00-624b-11eb-955b-65643d8dfd7e.png)
Please use this data
`data: [
  {value: 940, name: '搜索引擎'},
  {value: 660, name: '直接访问'},
  {value: 80, name: '邮件营销'},
  {value: 80, name: '联盟广告'}
]`

**Comment by weue** [comment]
> I cannot reproduce "the second label is disappear".

If you try to change the size of the browser window, you can reproduce it.

**Comment by Ovilia** [comment]
This was fixed in #14108, which is not released yet. It's scheduled in the next version.
